cfgov/unprocessed/css/on-demand/footer.less

```diff
@@ -10,16 +10,16 @@
 
 // Import required CF components.
 // Combined CF components.
-@import (less) 'cf-core.less';
-@import (less) 'cf-icons.less';
-@import (less) 'cf-buttons.less';
-@import (less) 'cf-forms.less';
+@import (less) '@cfpb/cfpb-core/src/cfpb-core.less';
+@import (less) '@cfpb/cfpb-icons/src/cfpb-icons.less';
+@import (less) '@cfpb/cfpb-buttons/src/cfpb-buttons.less';
+@import (less) '@cfpb/cfpb-forms/src/cfpb-forms.less';
 
 .o-footer {
-    @import (less) 'cf-buttons.less';
-    @import (less) 'cf-layout.less';
+    @import (less) '@cfpb/cfpb-buttons/src/cfpb-buttons.less';
+    @import (less) '@cfpb/cfpb-layout/src/cfpb-layout.less';
 }
```

Comment:
The nesting was so when you took the footer over to salesforce (I think was the use-case) its button styles wouldn't bleed over into the rest of the Salesforce app. But yeah the redundant buttons do appear like a mistake. Something for a different PR though. Not totally certain how to test because we manually copied the built footer over to salesforce.